File: competitive_programming/codeforces/06_javascript/4a_watermelon.js
function watermelon(weight) {
  return weight % 2 === 0 && weight > 2 ? "YES" : "NO";
}

if (require.main === module) {
  if (watermelon(8) !== "YES" || watermelon(2) !== "NO" || watermelon(3) !== "NO") throw new Error("watermelon failed");
  console.log("4A watermelon ok");
}

File: competitive_programming/codeforces/11_golang/4a_watermelon.go
package main

import "fmt"

func watermelon(weight int) string {
	if weight%2 == 0 && weight > 2 {
		return "YES"
	}
	return "NO"
}

func main() {
	if watermelon(8) != "YES" || watermelon(2) != "NO" || watermelon(3) != "NO" {
		panic("watermelon failed")
	}
	fmt.Println("4A watermelon ok")
}
